### PR TITLE
Changed text from SPI to ASWF in docs builder

### PR DESCRIPTION
**Pull request by rashil2000**
Signed-off-by: Rashil Gandhi <rashil2000@gmail.com>

### CODE REVIEW

File: docs/conf.py.in [hunk 1]
```diff
@@ -16,7 +16,7 @@ templates_path = ['templates']
 source_suffix = '.rst'
 master_doc = 'index'
 project = u'OpenColorIO'
-copyright = u'2010, Sony Pictures Imageworks'
+copyright = u'2020, Academy Software Foundation'
```
Comment (michdolan) [inline docs/conf.py.in]: I think the entire copyright statement should actually be changed to `Copyright Contributors to the OpenColorIO Project.` (with no year). That's what we use in sources. @jmertic can you confirm if this makes sense for the website, or should it be ASWF in that case?

File: docs/conf.py.in [hunk 2]
```diff
@@ -31,8 +31,8 @@ pygments_style = 'sphinx'
 rst_prolog = """
 .. |OCIO| replace:: *OCIO*
 .. |OpenColorIO| replace:: **OpenColorIO**
-.. _Sony Imageworks: http://www.imageworks.com/
-.. |Sony Imageworks| replace:: `Sony Imageworks`_
+.. _Academy Software Foundation: http://www.aswf.io/
```
Comment (michdolan) [inline docs/conf.py.in]: The URL should be: `https://www.aswf.io` (note: `http` -> `https`)

File: docs/conf.py.in [hunk 3]
```diff
@@ -85,21 +85,21 @@ latex_domain_indices = ['cpp-modindex', 'py-modindex']
...
 texinfo_appendices = []
 
 # -- Options for Epub output ---------------------------------------------------
 epub_title = u'OpenColorIO'
-epub_author = u'Sony Pictures Imageworks'
-epub_publisher = u'Sony Pictures Imageworks'
-epub_copyright = u'2010, Sony Pictures Imageworks'
+epub_author = u'Academy Software Foundation'
+epub_publisher = u'Academy Software Foundation'
+epub_copyright = u'2010, Academy Software Foundation'
```
Comment (michdolan) [inline docs/conf.py.in]: Same comment and question here about the copyright

File: docs/developers/getting_started.rst [hunk 4]
```diff
@@ -47,7 +47,7 @@ To check out a read-write version of the repository (GitHub acct required)::
     Receiving objects: 100% (2220/2220), 2.89 MiB | 2.29 MiB/s, done.
     Resolving deltas: 100% (1434/1434), done.
 
-Both read + read/write users should then add the Imageworks (SPI) master branch
+Both read + read/write users should then add the ASWF OpenImageIO master branch
```
Comment (michdolan) [inline docs/developers/getting_started.rst]: Change to `AcademySoftwareFoundation/OpenColorIO`, and I would suggest linking it to the GH page

File: docs/developers/submitting_changes.rst [hunk 5]
```diff
@@ -27,7 +27,7 @@ successfully contribute to the project. If any of the terms used are unfamiliar
 to you please do a quick search and then ask any of the contributors for
 assistance.
 
-* Fork the Imageworks OpenColorIO repository
+* Fork the ASWF OpenColorIO repository
```
Comment (michdolan) [inline docs/developers/submitting_changes.rst]: Let's change this to `AcademySoftwareFoundation/OpenColorIO` also, and link it to the GH page for clarity.

File: docs/conf.py.in (hunk 1 above)
Comment (jmertic) [reply docs/conf.py.in]: Yes, the website copyright should align with the same one aligned to for the codebase.

You might also want to consider a footer similar to OpenVDB - check their website https://openvdb.org for that language, which also includes the CC BY 40 license statement

**Pull request merged**